core/domain/exp_services.py

```diff
@@ -361,38 +469,72 @@ def get_exploration_ids_matching_query(query_string, cursor=None):
 def get_non_private_exploration_summaries():
     """Returns a dict with all non-private exploration summary domain objects,
     keyed by their id.
+
+    Returns:
+        dict. The keys are exploration ids and the values are corresponding 
+        non-private ExplorationSummary domain objects.
     """
     return _get_exploration_summaries_from_models(
         exp_models.ExpSummaryModel.get_non_private())
 
 
 def get_top_rated_exploration_summaries(limit):
-    """Returns a dict with top rated exploration summary domain objects,
+    """Returns a dict with top rated exploration summary model instances,
     keyed by their id.
+
+    Args:
+        limit: int. Number of exploration summary model instances to be
+            returned.
+
+    Returns:
+        dict. The keys are exploration ids and the values are corresponding
+        top rated ExplorationSummary domain model instances.
     """
     return _get_exploration_summaries_from_models(
         exp_models.ExpSummaryModel.get_top_rated(limit))
 
 
 def get_recently_published_exp_summaries(limit):
-    """Returns a dict with all featured exploration summary domain objects,
+    """Returns a dict with all featured exploration summary model instances,
     keyed by their id.
+
+    Args:
+        limit: int. Number of exploration summary model instances to be
+            returned.
+
+    Returns:
+        dict. The values are all featured ExplorationSummary model instances, 
+        keyed by their exploration id.
     """
     return _get_exploration_summaries_from_models(
         exp_models.ExpSummaryModel.get_recently_published(limit))
 
 
 def get_all_exploration_summaries():
-    """Returns a dict with all exploration summary domain objects,
+    """Returns a dict with all exploration summary model instanes,
```

Comment:
I think it will return ExplorationSummary domain objects , as because this function returns _ _get_exploration_summaries_from_models _which is defines as 
`def _get_exploration_summaries_from_models(exp_summary_models):
    """Returns a dict with ExplorationSummary domain objects as values, 
    keyed by their exploration id.

    Args:
        exp_summary_models: list(ExplorationSummary). List of ExplorationSummary
            model instances. 
    
    Returns:
        dict. The keys are exploration ids and the values are corresponding 
        ExplorationSummary domain objects.
    """`